index.html

```diff
@@ -922,7 +922,9 @@ <h4>Text</h4>
           </p>
           <p>A <a>Text</a> object is represented in a <a>DAPT Document</a> by a <code>&lt;p&gt;</code> element with the following constraints:</p>
             <ul>
-              <li>The <a>Text</a> of the <a>Script Event</a> is represented by the content of the <code>&lt;p&gt;</code> and its descendants, and after applying White Space Handling as defined in [[!XML]].
+              <li>The <a>Text</a> of the <a>Script Event</a> is represented by the character content
+              of the <code>&lt;p&gt;</code> and its descendants, and after applying White Space Handling
```

Comment:
Not sure if this is over-worrying, but is there a chance someone could parse this as "the character content of the `<p>` plus the (not necessarily character content) descendants of the `<p>`"?

Reply:
Would "and that of its descendants" work? If not, can you suggest alternative wording?